system_tests/fetch_and_run

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
why we need this? can't we use the file generated by the run of prep_env recipe?